rust/connlib/snownet/src/node.rs

```diff
@@ -502,7 +502,10 @@ where
         let conn = self
             .connections
             .get_established_mut(&cid)
-            .context(UnknownConnection(cid.to_string()))?;
+            .with_context(|| {
+                anyhow::Error::from(UnknownConnection(cid.to_string()))
+                    .context(format!("connection_id: {cid}"))
```

Comment:
Will be the first to admin I took a random stab here and not sure if this does what I'm after - adding the uniqueness fields as attributes instead of inline in the message